.github/workflows/gobuild.yml

```diff
@@ -3,33 +3,23 @@ name: Go Build
 on:
   push:
     branches: [main]
-    paths:
-      - '**.go'
-      - 'go.mod'
```

Comment:
Why was this removed?

Reply:
I'd rather keep filters in, because this build will run on every change when it does not need to - for example, if I change the Makefile then this will change.

Reply:
This PR was used for testing. Changes here are experimental and allowing us to replicate/check if we fixed problem.